src/mlpack/tests/serialization_test.cpp

```diff
@@ -38,7 +38,7 @@
 #include <mlpack/methods/lsh/lsh_search.hpp>
 #include <mlpack/methods/lars/lars.hpp>
 #include <mlpack/methods/bayesian_linear_regression/bayesian_linear_regression.hpp>
-#include <mlpack/methods/ann/rbm/rbm.hpp>
+//#include <mlpack/methods/ann/rbm/rbm.hpp>
```

Comment:
Another include that we could remove for now?

Reply:
I'll handle this after #3194; I want to split `serialization_test.cpp` such that serialization tests are done in the file relevant to the method.